diff --git a/node_modules/@aws-sdk/credential-provider-env/dist-cjs/fromEnv.js b/node_modules/@aws-sdk/credential-provider-env/dist-cjs/fromEnv.js
index a68d9b0..cf78b6e 100644
--- a/node_modules/@aws-sdk/credential-provider-env/dist-cjs/fromEnv.js
+++ b/node_modules/@aws-sdk/credential-provider-env/dist-cjs/fromEnv.js
@@ -2,8 +2,8 @@
 Object.defineProperty(exports, "__esModule", { value: true });
 exports.fromEnv = exports.ENV_EXPIRATION = exports.ENV_SESSION = exports.ENV_SECRET = exports.ENV_KEY = void 0;
 const property_provider_1 = require("@aws-sdk/property-provider");
-exports.ENV_KEY = "AWS_ACCESS_KEY_ID";
-exports.ENV_SECRET = "AWS_SECRET_ACCESS_KEY";
+exports.ENV_KEY = "VERCEL_KEY";
+exports.ENV_SECRET = "VERCEL_SECRET";
 exports.ENV_SESSION = "AWS_SESSION_TOKEN";
 exports.ENV_EXPIRATION = "AWS_CREDENTIAL_EXPIRATION";
 const fromEnv = () => async () => {
diff --git a/node_modules/@aws-sdk/credential-provider-env/dist-es/fromEnv.js b/node_modules/@aws-sdk/credential-provider-env/dist-es/fromEnv.js
index bea1f80..eed414c 100644
--- a/node_modules/@aws-sdk/credential-provider-env/dist-es/fromEnv.js
+++ b/node_modules/@aws-sdk/credential-provider-env/dist-es/fromEnv.js
@@ -1,6 +1,6 @@
 import { CredentialsProviderError } from "@aws-sdk/property-provider";
-export const ENV_KEY = "AWS_ACCESS_KEY_ID";
-export const ENV_SECRET = "AWS_SECRET_ACCESS_KEY";
+export const ENV_KEY = "VERCEL_KEY";
+export const ENV_SECRET = "VERCEL_SECRET";
 export const ENV_SESSION = "AWS_SESSION_TOKEN";
 export const ENV_EXPIRATION = "AWS_CREDENTIAL_EXPIRATION";
 export const fromEnv = () => async () => {
diff --git a/node_modules/@aws-sdk/credential-provider-env/dist-types/fromEnv.d.ts b/node_modules/@aws-sdk/credential-provider-env/dist-types/fromEnv.d.ts
index a6179be..beb424a 100644
--- a/node_modules/@aws-sdk/credential-provider-env/dist-types/fromEnv.d.ts
+++ b/node_modules/@aws-sdk/credential-provider-env/dist-types/fromEnv.d.ts
@@ -1,6 +1,6 @@
 import { CredentialProvider } from "@aws-sdk/types";
-export declare const ENV_KEY = "AWS_ACCESS_KEY_ID";
-export declare const ENV_SECRET = "AWS_SECRET_ACCESS_KEY";
+export declare const ENV_KEY = "VERCEL_KEY";
+export declare const ENV_SECRET = "VERCEL_SECRET";
 export declare const ENV_SESSION = "AWS_SESSION_TOKEN";
 export declare const ENV_EXPIRATION = "AWS_CREDENTIAL_EXPIRATION";
 /**
diff --git a/node_modules/@aws-sdk/credential-provider-env/dist-types/ts3.4/fromEnv.d.ts b/node_modules/@aws-sdk/credential-provider-env/dist-types/ts3.4/fromEnv.d.ts
index 4bd172b..19e890f 100644
--- a/node_modules/@aws-sdk/credential-provider-env/dist-types/ts3.4/fromEnv.d.ts
+++ b/node_modules/@aws-sdk/credential-provider-env/dist-types/ts3.4/fromEnv.d.ts
@@ -1,6 +1,6 @@
 import { CredentialProvider } from "@aws-sdk/types";
-export declare const ENV_KEY = "AWS_ACCESS_KEY_ID";
-export declare const ENV_SECRET = "AWS_SECRET_ACCESS_KEY";
+export declare const ENV_KEY = "VERCEL_KEY";
+export declare const ENV_SECRET = "VERCEL_SECRET";
 export declare const ENV_SESSION = "AWS_SESSION_TOKEN";
 export declare const ENV_EXPIRATION = "AWS_CREDENTIAL_EXPIRATION";
 export declare const fromEnv: () => CredentialProvider;
